demo_usage/cuboids_to_bboxes.py

```diff
@@ -122,37 +121,41 @@ def dump_clipped_3d_cuboids_to_images(
 
         city_name = dl.get_city_name(log_id)
         log_calib_data = dl.get_log_calibration_data(log_id)
-        ply_fpaths = dl.get_ordered_log_ply_fpaths(log_id)
+
         flag_done = False
-        for i, ply_fpath in enumerate(ply_fpaths):
-            if i % 50 == 0:
-                logging.info(f"\tOn file {i} of {log_id}")
-            lidar_timestamp = ply_fpath.split("/")[-1].split(".")[0].split("_")[-1]
-            lidar_timestamp = int(lidar_timestamp)
-            lidar_pts = load_ply(ply_fpath)
-
-            city_to_egovehicle_se3 = dl.get_city_to_egovehicle_se3(log_id, lidar_timestamp)
-            if city_to_egovehicle_se3 is None:
-                continue
-
-            labels = dl.get_labels_at_lidar_timestamp(log_id, lidar_timestamp)
-            if labels is None:
-                logger.info(f"\tLabels missing at t={lidar_timestamp}")
-                continue
-
-            for cam_idx, camera_name in enumerate(RING_CAMERA_LIST + STEREO_CAMERA_LIST):
-                camera_counter = 0
-                # load images, e.g. 'image_raw_ring_front_center_000000486.jpg'
-                im_fpath = dl.get_closest_im_fpath(log_id, camera_name, lidar_timestamp)
-                if im_fpath is None:
+        for cam_idx, camera_name in enumerate(RING_CAMERA_LIST + STEREO_CAMERA_LIST):
+            cam_im_fpaths = dl.get_ordered_log_cam_fpaths(log_id, camera_name)
+            for i, im_fpath in enumerate(cam_im_fpaths):
+                if i % 50 == 0:
+                    logging.info(f"\tOn file {i} of camera {camera_name} of {log_id}")
+
+                cam_timestamp = im_fpath.split("/")[-1].split(".")[0].split("_")[-1]
```

Comment:
Can we use `pathlib` or `os.path` to do this manipulation instead of string splitting?